go/test/endtoend/vtgate/queries/aggregation/aggregation_test.go

```diff
@@ -384,3 +384,16 @@ func TestAggregateRandom(t *testing.T) {
 
 	mcmp.AssertMatches("SELECT /*vt+ PLANNER=gen4 */ t1.shardKey, t1.name, count(t2.id) FROM t1 JOIN t2 ON t1.value != t2.shardKey GROUP BY t1.t1_id", `[[INT64(1) VARCHAR("name 1") INT64(2)] [INT64(2) VARCHAR("name 2") INT64(2)]]`)
 }
+
+func TestAggregateLeftJoin(t *testing.T) {
+	mcmp, closer := start(t)
```

Comment:
According to the PR's review checklist we should add a comment as to why this test was added and what it tests, even though the name is quite informative already.